### PR TITLE
Enable csp on lens proxy

### LINKED ISSUE #5576: Use Content-Security-Policy header
LensProxy should set CSP header (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy) that allows only `script-src 'unsafe-eval' 'self'; img-src *` by default. Distro builders should be able to provide custom CSP via package.json.

### PR DESCRIPTION
**Pull request by jakolehm**
Fixes #5576

### CODE REVIEW

File: src/main/lens-proxy/lens-proxy.ts [hunk 1]
```diff
@@ -239,6 +239,10 @@ export class LensProxy {
       }
     }
 
+    if (contentSecurityPolicy) {
+      res.setHeader("Content-Security-Policy", contentSecurityPolicy);
+    }
```
Comment (Nokel81) [inline src/main/lens-proxy/lens-proxy.ts:244]: Won't that always be defined?

Comment (jakolehm) [reply src/main/lens-proxy/lens-proxy.ts:244]: In theory it can be an empty string.

Comment (jakolehm) [reply src/main/lens-proxy/lens-proxy.ts:244]: Nevermind, it can't be.

File: src/common/vars.ts [hunk 2]
```diff
@@ -143,3 +143,4 @@ export const appSemVer = new SemVer(packageInfo.version);
 export const docsUrl = "https://docs.k8slens.dev/main/" as string;
 
 export const sentryDsn = packageInfo.config?.sentryDsn ?? "";
+export const contentSecurityPolicy = packageInfo.config?.contentSecurityPolicy || "script-src 'unsafe-eval' 'self'; frame-src http://*.localhost:*/; img-src *";
```
Comment (ixrock) [inline src/common/vars.ts]:
```suggestion
export const contentSecurityPolicy = packageInfo.config.contentSecurityPolicy;
```

Why not to keep defaults directly in `package.json`?

**Review by ixrock — DISMISSED** [review]

File: src/common/vars.ts (hunk 2 above)
Comment (jakolehm) [reply src/common/vars.ts]: @ixrock good point, will move defaults to package.json.

**Comment by jakolehm** [comment]
@Nokel81 @ixrock ptal again.

**Review by ixrock — APPROVED** [review]
LGTM

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**